src/main/java/org/apache/commons/crypto/utils/NativeCodeLoader.java

```diff
@@ -231,7 +231,7 @@ public static String getVersion() {
                 .getResource("/META-INF/maven/org.apache.commons.crypto/commons-crypto/pom.properties");
         if (versionFile == null) {
             versionFile = NativeCodeLoader.class
-                    .getResource("/org/apache/commons/crypto/VERSION");
```

Comment:
I've moved the VERSION file, so it is in the same package as the `NativeClassLoader` class. This way I don't need to declare the full path. Using the full path is basically the same in this case. If you like the full path for clarity, I can change this again.